api/src/main/java/com/debugmafia/clueless/state/Game.java

```diff
@@ -0,0 +1,144 @@
+package com.debugmafia.clueless.state;
+
+import com.debugmafia.clueless.actions.*;
+import com.debugmafia.clueless.model.*;
+
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.*;
+
+
+import javax.tools.DocumentationTool.Location;
+
+public class Game {
+    private HashSet<Card> winningCards;
+    private Board board;
+    private Turn turnInstance;
+    private List<Player> playerList;
+
+    public Game(Set<Player> players) {
+        this.board = new Board();
+        this.playerList = new ArrayList<>(players);
+        //TODO: Conversion above needs to have Mrs. Scarlett first
+        //TODO: Set the initialPlayers private List to the players
+        //TODO: Set the game state to GameState.IN_PROGRESS
+
+        this.winningCards = new HashSet<>(this.board.drawWinningCards());
+        //TODO: Determine how many cards go to each player (the number will always be a whole number),
+        //then call draw() for each player, giving them the cards that were drawn
+        
+        //TODO: Set the current players turn to the player who will go first, as well as other relevant turn
+        //information
+    }
```

Comment:
So for this constructor, I'd suggest splitting up the initialization logic into a few different `private` methods, which you can then call in here. That should make the logic more legible and less verbose.